tests/unit/routes/test_patient_router.py

```diff
@@ -66,6 +66,82 @@ def test_update_person(self, client):
         assert resp.json()["patient_reference_id"] == str(patient.reference_id)
         assert resp.json()["person_reference_id"] == str(new_person.reference_id)
 
+
+class TestCreatePatient:
+    def test_missing_data(self, client):
+        response = client.post("/patient")
+        assert response.status_code == 422
+
+    def test_invalid_person(self, client):
+        data = {"person_reference_id": str(uuid.uuid4()), "record": {}}
+        response = client.post("/patient", json=data)
+        assert response.status_code == 422
```

Comment:
```suggestion
        assert response.status_code == 422
        assert response.json() == {'detail': 'Person not found. Must include a person_reference_id that already exists in the MPI.'}

```

If more detailed errors are allowed.

Reply:
we sure can, I'm going to make some adjustments on the format as I'd like the detail response to match that of other 422 validation errors

```json
"detail": [
  {
    "loc": ["string"],
    "msg": "string",
    "type": "string"
  }
]
```